### DISCUSSION TITLE
What is the difference between "In-page Navigation" and "Side Navigation" components?

### DISCUSSION THREAD

**Discussion by jcklpe** [uswds/uswds#5995] [Q&A] [answered]
These seem pretty structurally similar to me:

<img width="991" alt="image" src="https://github.com/user-attachments/assets/5cbb768d-4cd4-4beb-bd5a-cd3a4aade5e7">
<img width="976" alt="image" src="https://github.com/user-attachments/assets/68c0cf79-b8d3-41d2-920d-7ca031022cda">

Seems like we could have some further guidance on when one is used versus the other, or perhaps even consolidation to a single component with variants.

**Comment by brunerae** [comment, marked as answer]
Hey @jcklpe - great question! The In-page navigation Component is used to navigate between section on a page, while the side navigation is used to navigate between different pages on a website.

**Reply by jcklpe** [reply]
Thanks! That makes sense. I think it might be helpful to include that information in the docs. 

Also just me being a monomaniac for simplification but it seems like the structure of these two components are very similar so I would be curious if it might be possible to consolidate styling some. For instance, they both use a lefthand sidebar to show the current User location but one has rounded corners and the other doesn't. One is a medium shade of blue, the other is a darker shade of blue. Are there particular reasons for these variations?

**Comment by ryparker** [comment]
The current guidance distinguishes the two by what they navigate:

- [Side navigation](https://designsystem.digital.gov/components/side-navigation/) represents a hierarchy of pages within a site section. Its current state identifies the page being viewed.
- [In-page navigation](https://designsystem.digital.gov/components/in-page-navigation/) links to headings within one long page and updates as the reader moves through it. Its guidance recommends a heading such as “On this page” and placement after the main content in the language's reading order.

Your follow-up about the different colors and corner shapes is a separate design question. The published guidance does not explain a specific rationale for those differences, so I would not claim they are required for accessibility. The components have different behavior, but that does not rule out sharing more of their visual styling. Consolidating that styling remains a design suggestion, rather than something the existing answer resolves.
